src/composables/acctVerfiedFunc.js

```diff
@@ -0,0 +1,7 @@
+import { auth } from 'firebase';
```

Comment:
this wouldn't give the type of output you think. As this is not the configured version of the application authentication. `getAuth(app)`

I would recommend you look into this code to make it a named export rather than a default export

Reply:
Alright

Reply:
I would push a configuration in the next hour